src/bomf/validation/core.py

```diff
@@ -241,7 +242,7 @@ def _narrow_supplied_dependencies(
             narrowed_depends_on.append(narrowed_dependency)
         return narrowed_depends_on
 
-    # pylint: disable=too-many-branches
+    # pylint: disable=too-many-branches,too-many-locals
```

Comment:
😢

Reply:
Ja, da könnte ich eventuell kleine Teile in Funktionen auslagern. Später ^^